art/classifiers/__init__.py

```diff
@@ -15,3 +15,4 @@
 from art.classifiers.catboost import CatBoostARTClassifier
 from art.classifiers.GPy import GPyGaussianProcessClassifier
 from art.classifiers.detector_classifier import DetectorClassifier
+from art.classifiers.classifier import ClassifierDecisionTree
```

Comment:
This is a good idea. Could you please append this import in line 5 above?

Reply:
of course!